layers/+spacemacs/spacemacs-purpose/packages.el

```diff
@@ -130,7 +130,8 @@
       (spacemacs/set-leader-keys
         "rb" 'purpose-switch-buffer-with-purpose
         "rB" 'switch-buffer-without-purpose
-        "rd" 'purpose-toggle-window-purpose-dedicated
+        "rdb" 'purpose-toggle-window-buffer-dedicated
+        "rdw" 'purpose-toggle-window-purpose-dedicated
```

Comment:
I think the `rd` prefix should be labeled somewhere so that which-key shows it with something more informative than `prefix`